libr/asm/p/asm_x86_nz.c

```diff
@@ -2466,6 +2466,10 @@ static int oppush(RAsm *a, ut8 *data, const Opcode *op) {
 			}
 		}
 	} else {
```

Comment:
Thats a compile time check. You should check the immediate value instead of its size